ImageUtil.getResourceStreamFromClass does not log attempted path if load fails

As it currently stands, the method will (after a lengthy set of stacktraces) yield the following exceptions:

```
Caused by: java.lang.IllegalArgumentException: file.png
 
Caused by: java.lang.IllegalArgumentException: input == null!
```

It would be helpful to developers to print the searched path so they can more easily figure out why their images are not being loaded.

--- a/runelite-client/src/main/java/net/runelite/client/util/ImageUtil.java
+++ b/runelite-client/src/main/java/net/runelite/client/util/ImageUtil.java
@@ -351,15 +351,18 @@ public class ImageUtil
 		}
 		catch (IllegalArgumentException e)
 		{
+			String filePath;
+
 			if (path.startsWith("/"))
 			{
-				log.warn("Failed to load image from class: {}, path: {}", c.getName(), path);
+				filePath = path;
 			}
 			else
 			{
-				log.warn("Failed to load image from class: {}, path: {}",
-						c.getName(), c.getPackage().getName().replace(".", "/") + "/" + path);
+				filePath = c.getPackage().getName().replace(".", "/") + "/" + path;
 			}
+
+			log.warn("Failed to load image from class: {}, path: {}", c.getName(), filePath);
 
 			throw new IllegalArgumentException(path, e);
 		}

--- a/runelite-client/src/main/java/net/runelite/client/util/ImageUtil.java
+++ b/runelite-client/src/main/java/net/runelite/client/util/ImageUtil.java
@@ -351,6 +351,8 @@ public class ImageUtil
 		}
 		catch (IllegalArgumentException e)
 		{
+			String folderPath = System.getProperty("user.dir") + "\\resources\\plugins\\" + c.getPackage().getName();
+			log.error("Failed to load image from class: {}, path: {}", c.getName(), folderPath);
 			throw new IllegalArgumentException(path, e);
 		}
 		catch (IOException e)

--- a/runelite-client/src/main/java/net/runelite/client/util/ImageUtil.java
+++ b/runelite-client/src/main/java/net/runelite/client/util/ImageUtil.java
@@ -351,8 +351,16 @@ public class ImageUtil
 		}
 		catch (IllegalArgumentException e)
 		{
-			String folderPath = System.getProperty("user.dir") + "\\resources\\plugins\\" + c.getPackage().getName();
-			log.error("Failed to load image from class: {}, path: {}", c.getName(), folderPath);
+			if (path.startsWith("/"))
+			{
+				log.warn("Failed to load image from class: {}, path: {}", c.getName(), path);
+			}
+			else
+			{
+				log.warn("Failed to load image from class: {}, path: {}",
+						c.getName(), c.getPackage().getName().replace(".", "/") + "/" + path);
+			}
+
 			throw new IllegalArgumentException(path, e);
 		}
 		catch (IOException e)

--- a/runelite-client/src/main/java/net/runelite/client/util/ImageUtil.java
+++ b/runelite-client/src/main/java/net/runelite/client/util/ImageUtil.java
@@ -351,7 +351,7 @@ public class ImageUtil
 		}
 		catch (IllegalArgumentException e)
 		{
-			String filePath;
+			final String filePath;
 
 			if (path.startsWith("/"))
 			{

